src/collections/_documentation/platforms/php/index.md

```diff
@@ -382,12 +422,19 @@ use Sentry\State\Hub;
 
 $options = ['dsn' => '___PUBLIC_DSN___'];
 
-$spool = new MemorySpool();
-$transport = new SpoolTransport($spool);
 $httpTransport = new HttpTransport($options, HttpAsyncClientDiscovery::find(), MessageFactoryDiscovery::find());
 
+$spool = new MemorySpool();
+
+$transportFactory = new class implements TransportFactoryInterface {
+    public function create(\Sentry\Options $options): \Sentry\Transport\TransportInterface
+    {
+        return new SpoolTransport($spool);
+    }
+};
+
 $builder = ClientBuilder::create($options);
-$builder->setTransport($transport);
+$builder->setTransportFactory($transportFactory);
```

Comment:
Somewhere after this call there should be a call to `$spool->flushQueue($httpTransport);`. This means that you should instantiate another transport (e.g. the HTTP transport) yourself and use it here

Reply:
It's there, 3 lines below.

Reply:
Ops, I didn't see it